jenkins-pipelines/manager/centos-manager-backup-azure.jenkinsfile

```diff
@@ -5,11 +5,14 @@ def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
 
 managerPipeline(
     backup_bucket_backend: 'azure',
-    backend: 'aws',
+    backend: 'azure',
     ip_ssh_connections: 'public',
-    region: '''["us-east-1", "us-west-2"]''',
+    azure_region_name: 'eastus',  //TODO: modify to multy region once sct supports more than one region
```

Comment:
why do you need it here, and in the `.yaml` file?

Reply:
Removed from yaml